### PR TITLE
The code for Bubble Sort algorithm was added

### LINKED ISSUE #3: Add a Python program for Bubble Sort Algoritm
Definition of done:
1. Clear code for Bubble Sort Algoritm in python should be written.
2. Comments will be really appreciable.
3. Add minimum 2 test cases to test the code.
4. Add you code under Python/Data Structure and Algorithms/

### PR DESCRIPTION
**Pull request by Vaani-pathariya**
The code and explanation for Bubble sort algorithm along with test cases was added 
Looking forward to get it merged !
fixes #3

### CODE REVIEW

File: Python/Data Structure and Algorithms/BubbleSort.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    for i in range (len(arr)):
+        print(arr[i],end=" ")
+
+#Test Case 1: 
+array1=[64, 34, 25, 12, 22, 11, 90]
+
+#Test Case 2:
+array2=[332, 754, 23, 1, 0, 1, 645, 34]
+
+#Change the array in function calls from array1 to array2 to see the output for the second test case
```
Comment (jatinkumar027) [inline Python/Data Structure and Algorithms/BubbleSort.py:41]: Test Case 2 is never called.

**Comment by jatinkumar027** [comment]
Can you attach any terminal output for this?

**Comment by Vaani-pathariya** [comment]
For array 1 : 
image
For array 2:
image

**Review by jatinkumar027 — COMMENTED** [review]
approved

**Pull request merged**